content/articles/introduction-to-secrets-management-with-doppler/index.md

```diff
@@ -0,0 +1,179 @@
+
+![Connecting to Doppler Repo](/engineering-education/introduction-to-secrets-management-with-doppler/scoop.png)
+
+After the Doppler bucket is successfully added, you'll install Doppler CLI through the script below:
+
+```bash
+scoop install doppler
+```
+
+![Installing Doppler](/engineering-education/introduction-to-secrets-management-with-doppler/cmd-doppler.png)
+
+You can initiate the authentication process when the installations are complete by running `doppler` login within your command prompt to enter your Doppler account details. 
+
+#### Doppler dashboard
+Doppler dashboard is a browser-based interactive platform where users can organize secrets into projects and environments. The Doppler Dashboard has access to secrets in your projects, keeping the local development and Doppler in sync.
+
+#### Working with Doppler
+Within Doppler, you can create as many projects as you need - considering the applications being developed. The secrets in every project can exist in any environment – development, test, load, or production.
+
+![Doppler dashboard](/engineering-education/introduction-to-secrets-management-with-doppler/project.png)
+
+Now, it's time to create a project that will handle secrets.
+
+Navigate the Doppler dashboard to the `+` button and click to launch a project. You can name your project to reflect what secrets you'll be storing in your workspace. 
+
+![Creating a Project](/engineering-education/introduction-to-secrets-management-with-doppler/new-project.png)
+
+Any project created in Doppler comes with dev, test, and production environments to help define secret levels.
+
+![Project Environment](/engineering-education/introduction-to-secrets-management-with-doppler/project_env.png)
+
+You can then store secrets singly or import some JSON files with secrets - creating a bulk secrets option.
+
+To do this, click the `Add Secret` button that injects a secret key-value into your project for saving.
+
+![Secrets Page](/engineering-education/introduction-to-secrets-management-with-doppler/secrets.png)
+
+![URL Secret Example](/engineering-education/introduction-to-secrets-management-with-doppler/secret-example.png)
+
+You can connect the local project environment to your Doppler secrets through the Doppler CLI. Navigate to the existing local project folder within your terminal or code editor and run the snippet below.
+
+The command will set up your current Doppler project in the local development environment:
+
+```bash
+doppler setup 
+``` 
+
+![Set Up](/engineering-education/introduction-to-secrets-management-with-doppler/local_configs.png)
+
+After connecting your local project with the Doppler project, you can access, filter, and download secret(s) as either plain values or JSON files.
+
+Run the command `doppler secrets get DOPPLER_PROJECT DOPPLER_CONFIG --plain` to select your current project and its environment, as shown below:
+
+![Project](/engineering-education/introduction-to-secrets-management-with-doppler/get.png)
```

Comment:
We don't have this image. Please add it. Make sure the extensions are in lowercase